src/components/UI/dropdown_menu.js

```diff
@@ -47,7 +66,7 @@ function DropdownMenu(props) {
 
   const onBlurHandler = () => {
     timeOutId = setTimeout(() => {
-      setIsOpen(false);
+      // setIsOpen(false);
```

Comment:
As a codestyle thing, I think we should avoid commented code in the codebase .

Is the surrounding code and blurhandler still relevant?

Reply:
Good point. Yes, I should have uncommented it